partner_address_two_lines/models/res_partner.py

```diff
@@ -1,16 +1,40 @@
-# Copyright 2020 Camptocamp SA
+# Copyright 2024 Camptocamp SA
```

Comment:
Don't change it
![image](https://github.com/trisdoan/partner-contact/assets/8724793/42b8ddb9-4cf2-4222-b93c-517f7173da14)